content/analyst/competitive-tracker/brand-tracker.md

```diff
@@ -0,0 +1,87 @@
+
+**Headquarters -** Displays the physical address of the brand headquarters, which is sometimes the same address as the parent company
+
+**Industry -** Displays the industry that the brand is associated with such as “Apparel & Accessories” or “Restaurants, Bars & Food Service”
+
+ Consider that different brands within the same Company may have different industries. For example, Tide Detergent is associated with “Cleaning Services & Supplies” whereas Pepto Bismol is associated with “Drugs & Vaccines” but they’re both within The Procter & Gamble Co. company.
+
+**ESPs -** Displays the Email Service Providers (ESPs) the brand’s domains are sending from
+
+**Email Volume by Domain -** Displays the volume of the top five volume domains from the previous week
+
+**Engagement Stats by Domain -** Displays the read rate of the top five volume domains from the previous week
+
+**Competitors -** Displays the domains and companies of the top competitors within the industry
+
+**Average Campaigns per Week -** Displays the average number of campaigns per week from the previous four weeks
+
+**Most Active Day -** Displays the most active day of the week from the previous four weeks
+
+**Top Volume Domains -** Displays the top volume domains, sorted by the average volume of recipients per day from the previous four weeks
+
+**Top Performing Domains -** Displays the top performing domains, sorted by their performance score from the previous four weeks
+
+ The top performers is a proprietary performance score based upon the sender’s volume, inbox placement, and engagement metrics.
+
+**Top Sending Partners -** Displays the number of campaigns from the last four weeks that have driven traffic to the domains associated with this brand
+
+ If you click on the number of campaigns, you will be directed to an Advanced Search which will list all of the campaigns with links to this brand’s domains. This data is calculated based on campaigns received by our research team’s Virtual Characters.
+
+**Latest Campaigns -** Displays the campaigns received by our Competitive Tracker data sources in the last 30 days, and is available in both List Mode and Visual Mode:
+
+*List Mode* displays the campaign’s Latest Sent date, Subject line, Volume, Inbox Placement, Spam Rate, Read Rate, Read+Deleted Rate, and Deleted Rate. When available, it also displays the campaign’s Target Country, as well as links to the campaign’s Creative, Raw HTML Headers, and [Journey Mapping](/analyst/competitive-tracker/journey-mapping-in-competitive-tracker) 
+
+*Visual Mode* displays the same information but highlights the creatives for each campaign.
+
+ We redact all identifying features of an email to protect our [panelists’ privacy](/analyst/privacy-gdpr/consumer-panel-data-collection-privacy-practices) . For example, the person’s name, shipping address, products ordered, etc. will either be replaced with “[REDACTED]” in List Mode or blurred out in Visual Mode.
```

Comment:
`/analyst/privacy-gdpr/consumer-panel-data-collection-privacy-practices` -> `/analyst/privacy-gdpr/consumer-panel-data-collection-and-privacy-practices/`